dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -201,5 +210,34 @@ def self.filtered_labels(course)
     def self.can_see_locked_status?(user)
       user && (user.workshop_admin? || user.regional_partners.first.try(&:group) == 3)
     end
+
+    def find_default_workshop
```

Comment:
You've moved this functionality and removed all the old tests for it, but I don't see any new tests for it.  Can we repurpose any of the removed unit tests to cover this business logic again?